lib/lib_rss.php

```diff
@@ -565,8 +571,11 @@ function httpGet(string $url, string $cachePath, string $type = 'html', array $a
 		// TODO: Implement HTTP 410 Gone
 	} elseif (!is_string($body) || strlen($body) === 0) {
 		$body = '';
-	} elseif ($type !== 'json') {
-		$body = enforceHttpEncoding($body, $c_content_type);
+	} else {
+		$body = trim($body, " \n\r\t");
```

Comment:
Is overriding the defaults presumed to be better for performance?

Reply:
Do you mean compared to letting DOMDocument decide, or what overriding more precisely?
In any case, some overriding is needed to convey the HTTP charset information

Reply:
No, I mean compared to simply:
```suggestion
		$body = trim($body);
```

Reply:
Ah, for `trim()`, alright, I was afraid for breaking some [encoding BOMs](https://en.wikipedia.org/wiki/Byte_order_mark)

Reply:
Ah, you mean like UTF-32 or something? Maybe a quick comment to that effect would be useful.

Reply:
UTF-32BE starts with `\00`, for instance. But vertical tab does not seem to be a problem after all

Reply:
Done